ubuntu_package_changelog/__init__.py

```diff
@@ -104,22 +121,115 @@ def main():
             'ubuntu-package-changelog',
             'production', version='devel')
 
-    changelog_url = _lp_get_changelog_url(args, lp)
-    if not changelog_url:
+    sources = _lp_get_sources(args, lp)
+
+    if len(sources) != 1:
         print('no changelog found')
         sys.exit(0)
 
+    changelog_url = _lp_get_changelog_url(sources[0])
+    source_package_name = sources[0].source_package_name
+
     url = lp._root_uri.append(urllib.parse.urlparse(changelog_url).path.lstrip('/'))
     resp = lp._browser.get(url).decode('utf-8')
     entry_count = 0
+    changelog_entry_lines = []
     for line in resp.splitlines():
         line = line.rstrip()
+
+        # this is the first line of a changelog entry. Find the version and reset lists for
+        # changelog entry lines and changelog entry cves
+        if source_package_name in line and line.startswith(source_package_name) and ' (' in line and ')' in line:
```

Comment:
can we use the `python-debian` module to do that? then we would get a list of changes from that module. eg:
```
c = debian.changelog.Changelog(open('/home/tom/tmp/changelog'))
```
I also wonder if we should add a method to that module to get a list of CVEs (similar to the method which provides a list of closed LP bugs:
```
>>> c[0].lp_bugs_closed
[1989194, 1993204]
```